release.yaml

```diff
@@ -35,6 +35,8 @@ requirements:
   "istioMinimalVersion": "1.16" # modules/110-istio/requirements/check.go
   "yandexHasDeprecatedZoneInConfig": "false" # modules/030-cloud-provider-yandex/requirements/check.go
   "yandexHasDeprecatedZoneInNodes": "false" # modules/030-cloud-provider-yandex/requirements/check.go
+  "l2LoadBalancerModuleEnabled": "false" # ee/modules/381-l2-load-balancer/requirements/check.go
```

Comment:
we have the `disabledModules` [key](https://github.com/deckhouse/deckhouse/pull/9658/files#diff-a0b29149f10adefb5c44814777ca3aa3c992d81fd92a1a051add31fab99452fcR27) in this file, I guess it's better to use it.

Reply:
Fixed

Reply:
Probably, you don't need this key. But it's up to you